### DISCUSSION TITLE
Syntax Highlighting in VS Code

### DISCUSSION THREAD

**Discussion by mattbrannon** [mdx-js/mdx#2321] [Q&A] [answered]
I installed the [MDX](https://marketplace.visualstudio.com/items?itemName=unifiedjs.vscode-mdx) extension for VS Code. However, when working with a `.mdx` file, syntax highlighting is broken anytime text is put between opening and closing tags on the same line.

<img width="706" alt="Screenshot 2023-07-05 at 12 24 24 AM" src="https://github.com/mdx-js/mdx/assets/36570183/dee492a3-d965-435f-be09-22fc05afc432">

Is there something I need to do beyond installing the extension to get this work?

**Comment by wooorm** [comment, marked as answer]
Hi. I think this was https://github.com/wooorm/markdown-tm-language/commit/82f6319ff54bae849a8252851cfb2d902658bc6c or at least it’s not a problem in the grammar file on its own https://wooorm.com/markdown-tm-language/. At some point in the future a new release of the VS Code extension should do it (ref https://github.com/mdx-js/mdx-analyzer/issues/326#issuecomment-1602473304)

**Reply by mattbrannon** [reply]
Thanks for the quick reply. Just wanted to make sure I wasn't missing something. Looking forward to the new release.
